docs/public/samples/z-up.html

```diff
@@ -56,18 +56,23 @@
       document.body.appendChild(renderer.domElement);
 
       const [threeModel, threeModelAnimation] = loadThreeModel(scene, renderer);
-      // Adjust rotation for the z-up
+      // Adjust rotations for the Z-up coordinate system
       threeModel.rotation.x = Math.PI / 2;
-      threeModel.rotation.y = Math.PI;
       scene.add(threeModel);
 
+      // Init OrbitControls first
+      const controls = new OrbitControls(camera, renderer.domElement);
```

Comment:
It's necessary to initialize the orbit controls before the viewport to ensure the viewport instantiates with a camera "controlled" by the orbit controls. Otherwise the Gizmo is not initialized to the correctly controlled orientation for Z-up and X-up systems.